activemq/datadog_checks/activemq/data/conf.yaml.example

```diff
@@ -40,12 +40,12 @@ instances:
     ## @param host - string - required
     ## JMX hostname to connect to.
     #
-  - host: <HOST>
+  - host: localhost
```

Comment:
Actually, we don't need to override this anymore don't we? host and port are overriden in `conftest`.

The placeholder are our usual way to proceed for required parameters, any reason to change them?

Reply:
> Actually, we don't need to override this anymore don't we? 
Overriding in the test makes the test more independent to the sample config.

> The placeholder are our usual way to proceed for required parameters, any reason to change them?
The sample files had these values before applying specs. I'm happy to revert this override but I thought is probably useful to keep these default values